docs/iot/iot-partners/index.md

```diff
@@ -0,0 +1,27 @@
+# Tizen IoT with Nubison Cloud
+
+The Nubison cloud can readily be interconnected with Tizen IoT devices, which has an intrinsic capability of seamless connectivity.
+Real-time data monitoring and controlling between Tizen service app and Nubison cloud is empowered through Nubison plugin.
+
+**Figure: Architecture**
+
+![Architecture](media/architecture.png)
+
+Compatibility
+- Platform : Tizen 5.0 IoT Headed & Headless and later
+- SDK : Tizen Studio 3.0 and later 
+
+
+## What is Nubison IoT?
+Nubison IoT provides various IoT features and services which can be utilized in different IoT devices. Data analysis service is one of distinguished IoT services that Nubison provides.
+
+
+References
+- [Github for Nubison Template](https://github.com/nubisoniot/NI)
+- [Nubison Site for Developers](https://nubisoniot.com/)
+- [Nubison: Your IoT Service Provider](https://blog.naver.com/nubison/221462430388)
+- [How to use Nubison IoT](https://blog.naver.com/nubison/221462229766)
```

Comment:
For more information about Nusion IoT cloud, see the following references:

- [Github for Nubison Template](https://github.com/nubisoniot/NI)
- [Nubison Site for Developers](https://nubisoniot.com/)
- [Nubison: Your IoT Service Provider](https://blog.naver.com/nubison/221462430388)
- [How to use Nubison IoT](https://blog.naver.com/nubison/221462229766)

Reply:
@bson1012 kindly update the text